Remotion/Web/Development.WebTesting/HostingStrategies/AspNetCoreHostingStrategy.cs

```diff
@@ -63,10 +69,12 @@ public void DeployAndStartWebApplication ()
           "The executable's path was not set while trying to host with ASP.NET Core. Please add the 'processPath' attribute to the 'testSiteLayout' configuration.");
     }
 
+    var http = _useHttps ? "https" : "http";
```

Comment:
We could use `protocol` as variable name here instead. That would fit `http` and `https`